subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -284,6 +284,31 @@ config DEVICE_NAME_GATT_WRITABLE_AUTHEN
 
 endif #BT_DEVICE_NAME_GATT_WRITABLE
 
+config BT_DEVICE_APPEARANCE_GATT_WRITABLE
+	bool "Allow to set GAP Appearance by remote GATT clients"
+	depends on BT_DEVICE_APPEARANCE_DYNAMIC
+	default y
+	help
+	  Enabling this option allows remote GATT clients to write to device
+	  appearance GAP characteristic.
+
+if BT_DEVICE_APPEARANCE_GATT_WRITABLE
+	config BT_DEVICE_APPEARANCE_GATT_WRITABLE_ENCRYPT
+	bool "Encryption required to write appearance by remote GATT clients"
+	default y
+	help
+	  Enabling this option requires the connection to be encrypted to write
+	  to the device appearance GAP characteristic.
+
+
+config DEVICE_APPEARANCE_GATT_WRITABLE_AUTHEN
+	bool "Authentication required to write appearance by remote GATT clients"
+	help
+	  Enabling this option requires the connection to be encrypted and
+	  authenticated to write to the device appearance GAP characteristic.
```

Comment:
Should `DEVICE_APPEARANCE_GATT_WRITABLE_AUTHEN` depend on `BT_DEVICE_APPEARANCE_GATT_WRITABLE`? Or should these be a `choice`? 

I see that it's following the name above, so I guess it's OK and would otherwise need to be fixed for both. 

Since you are modifying the default of the characteristic (going from authenticated to just encryption), we should mentioned that in the release notes, or default to the same value

Reply:
I kept this consistent with name,  didn't wanted to over-engineer this, also I can keep authenticated as default